src/marqo/core/index_management/index_management.py

```diff
@@ -96,12 +98,17 @@ def bootstrap_vespa(self) -> bool:
         app = self.vespa_client.download_application()
         configured = self._marqo_config_exists(app)
 
+        query_profile_updated = self._update_query_profile(app)
+
         if not configured:
             self._add_marqo_config(app)
             self.vespa_client.deploy_application(app)
             self.vespa_client.wait_for_application_convergence()
             self._save_marqo_version(version.get_version())
             return True
+        elif query_profile_updated:
+            self.vespa_client.deploy_application(app)
+            self.vespa_client.wait_for_application_convergence()
```

Comment:
We might want to return True here as well, since we redeployed the application. What are the implications of this?

Reply:
ah yes I missed this. we should return True